lib/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -161,6 +172,60 @@ public UserIdentity generateUserIdentity(
         return userIdentityBuilder.build();
     }
 
+    private JsonNode getVCClaimNode(String credential) throws HttpResponseExceptionWithErrorBody {
```

Comment:
I did some code refactor for duplicate codes in here. But, We need to more refactor old some methods after. Because, We need to sure with all changes test.  We can use above generic methods for them.